Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -21,6 +21,7 @@
 #import "SPObjectManager.h"
 #import "SPAddCollaboratorsViewController.h"
 #import "JSONKit+Simplenote.h"
+#import "DTPinLockController.h"
```

Comment:
So true! I accidentally left it in there from some earlier work. Removed in fc572dd